lib/common.ml

```diff
@@ -170,3 +170,5 @@ module Re2 = struct
   let wrap s = create_exn s
   let unwrap = Re2.to_string
 end
+
+let ( let* ) = Lwt_result.bind
```

Comment:
i would move this to the top of the file. It's where you'd expect to find this kind of stuff